src/api.ts

```diff
@@ -96,6 +96,11 @@ export class JamsocketApi {
     return new JamsocketApi(apiBase, { rejectUnauthorized })
   }
 
+  public getLoginUrl(): string {
+    const hostname = new URL(this.apiBase).hostname
+    return `https://app.${hostname}/cli-login`
```

Comment:
Seems like there should be a better way to define this 🤔

Reply:
I think this is fine for now, because in a future iteration I think we'll want to do this a bit differently:

- Hit an endpoint on our side that returns back two URLs: a server-sent events URL that the client listens to, and a URL to direct the user to. Both would contain a unique token that pairs them on the backend.
- When the user completes the log-in, our backend sends an event over the SSE connection that includes the token, so the user never has to copy/paste anything.